docs/en_US/install/wiiu.md

```diff
@@ -90,11 +90,15 @@ You will want to register an account from [here](/account) and click `Don't have
 
 <img src="/assets/images/docs/install/wiiu/pnid/register-account-web.png" width="100%">
 
-Right before registering a PNID, please make sure to save your password on some sort of note, as you will not be able to reset your password. After registering just simply change your birth date, gender, timezone, country/region, etc to what you see fit. Once you have it set up we can switch back to the Wii U. Within your Wii U you will want to reconnect to Pretendo _**Note:** how you reconnect to Pretendo depends on how you installed it on your Wii U!_ Once you are reconnected, you will want to press the top left profile icon and press `Switch Account`, then press `Add New User`. Go through the user set up normally, but when it asks if you have an account, press `Yes` and put in your username and password. If it asks you to confirm your email afterwards, simply skip it. You will now be able to use Pretendo Servers
+Right before registering a PNID, please make sure to save your password on some sort of note, as you will not be able to reset your password. After registering just simply change your birth date, gender, timezone, country/region, etc to what you see fit. Once you have it set up we can switch back to the Wii U. Within your Wii U you will want to reconnect to Pretendo. 
```

Comment:
While we're here, password resetting has been implemented for a long time. So the section about it not being implemented is no longer needed